backend/company-agent/src/main/java/org/hyperledger/oa/impl/aries/SchemaService.java

```diff
@@ -85,26 +92,37 @@ public SchemaAPI addSchema(@NonNull String schemaId, @Nullable String label) {
 
     public List<SchemaAPI> listSchemas() {
         List<SchemaAPI> result = new ArrayList<>();
-        schemaRepo.findAll().forEach(dbS -> result.add(SchemaAPI.from(dbS)));
+        schemaRepo.findAll().forEach(dbS -> {
+            SchemaAPI schemaAPI = SchemaAPI.from(dbS);
+            schemaAPI.addSchemaAttributes(getSchemaAttributeNames(dbS.getSchemaId()));
```

Comment:
getSchemaAttr() is a web request. I would not do this on each read, I would do this once when saving the new schema and persist the attributes with the schema, because a schema will never change again.